src/main/webapp/js/courseSelector.js

```diff
@@ -148,27 +153,28 @@ export const CollegePlanner = (() => {
     deleteButtonElement.setAttribute(
         'class', 'float-right rounded-circle border-0');
     deleteButtonElement.addEventListener('click', () => {
+      Calendar.removeCourse(liElement.getAttribute('value'));
```

Comment:
this might cause an error for the planner page that also depends on this file -- is there possible way to add this event listener in the calendar.js file?

Reply:
What would cause the error? The `Calendar` namespace?

Reply:
When calling that function, it would try to modify elements that don't exist in the other page